Automattic/jetpack
# Social: Fix share status endpoint call feature check

Fixes https://github.com/Automattic/jetpack-reach/issues/570

## Proposed changes:

* Return early if the feature flag is off

### Other information:


## Jetpack product discussion


https://github.com/Automattic/jetpack-reach/issues/570
## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






* Go to the editor and open the jetpack tab
* Pay attention to the `share-status` endpoint call in the network tab
* Turn off the feature flag, do the first 2 steps, the call should not show up
* Do the same on Simple and Atomic sites

![image](https://github.com/user-attachments/assets/497959e8-2d52-469f-bdb6-84d6b1352dcd)
